wire-ios/Wire-iOS/Sources/UserInterface/Settings/DebugCommand.swift

```diff
@@ -40,6 +45,9 @@ enum DebugCommand {
         case "resync resources":
             self = .resyncResources
 
+        case "initiateResetMLSGroups":
```

Comment:
nit-pic: keep consistent naming with other commands
```suggestion
        case "initiate reset mls groups":
```

Reply:
skipping this for now